docs/USAGE-GUIDE.md

```diff
@@ -535,6 +544,58 @@ underlying encrpyt/decrypt functions are not available in older versions.
 3. Prefer encryption ciphers in the following order: AES128, AES256, ChaCha20, 3DES, RC4.
 4. Prefer record authentication modes in the following order: GCM, Poly1305, SHA256, SHA1, MD5.
 
+The following chart maps the security policy version to the signature scheme supported:
+
+|    version     |   RSA PKCS1  | ECDSA - TLS 1.2 |  SHA-1 Legacy |  RSA PSS | 
```

Comment:
What about ECDSA - TLS 1.3?

Reply:
ECDSA - TLS 1.3 is not a part of the signature scheme s2n_sig_scheme_pref_list_20140601 : https://github.com/awslabs/s2n/blob/master/tls/s2n_signature_scheme.c#L187-L205 or s2n_sig_scheme_pref_list_20200207 https://github.com/awslabs/s2n/blob/master/tls/s2n_signature_scheme.c#L208-L234

Reply:
"ECDSA - TLS 1.3" would be "s2n_ecdsa_secp256r1_sha256" and "s2n_ecdsa_secp384r1_sha384" from those lists. TLS1.3 specifies curve.

But also, do we really need to distinguish between the two? Can't we just say "ECDSA"?

Reply:
We do not require to distinguish between the two, will update to `ECDSA`.